src/git/git-service.js

```diff
@@ -40,6 +42,32 @@ internals.getPullRequest = (github, repo_owner, repo_name, pull_request_number)
   });
 };
 
+internals.getOpenPullRequestForSpecificBranch = (github, repo_owner, repo_name, branch_name) => {
+  return new Promise((resolve, reject) => {
+    github.pullRequests.getAll({
+      user: repo_owner,
+      repo: repo_name,
+      state: 'open',
+      head: branch_name,
+    }, (err, res) => {
+      if (err) {
+        reject(err);
+        return;
+      }
+
+      if (res && Array.isArray(res) && res.length > 0) {
```

Comment:
FYI, `Array.isArray(res)` returns `false` if `res` wouldn't exist too. Hence, the first check is redundant.

(No need to change.)